fighters/link/src/acmd/aerials.rs

```diff
@@ -6,28 +6,26 @@ unsafe fn attack_air_n(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
     frame(lua_state, 1.0);
-    if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 0.429);
-    }
+    FT_MOTION_RATE(fighter, 0.5);
```

Comment:
lets change this to the standard `FT_MOTION_RATE` format `FT_MOTION_RATE(fighter, C / (B - A))`